ansible/roles/openregister_config/templates/openregister-config.yaml.j2

```diff
@@ -58,6 +58,10 @@ downloadConfigs: true
 
 server:
   registerDefaultExceptionMappers: false
+  requestLog:
+    type: classic
```

Comment:
I cant see anything about this type attribute in the documentation. Whats it for?